Makefile

```diff
@@ -92,11 +92,12 @@ metering-ansible-operator-docker-build: $(METERING_ANSIBLE_OPERATOR_DOCKERFILE)
 
 # Runs gofmt on all files in project except vendored source
 fmt:
+	@echo path: $(shell pwd)
 	find . -name '*.go' -not -path "./vendor/*" | xargs gofmt -w
 
 # Update dependencies
-vendor: Gopkg.toml
-	dep ensure -v
+vendor:
+	go mod vendor
```

Comment:
```suggestion
	go mod tidy
	go mod vendor
	go mod verify
```

Reply:
Not sure we need explicit calls to `go mod tidy` or `go mod verify` because eventually we call `go build` which runs both those with `go.mod` present, right?

Reply:
I'm also thinking we may want to build binaries using `go build -mod=vendor`.

Reply:
Ok after reading more here:
https://github.com/golang/go/wiki/Modules#how-to-prepare-for-a-release (really good write-up of releasing with Go modules, I hadn't found before) I believe go mod tidy is needed to removed unneeded deps.
And after reading more here: https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away -- rationale about committing the go.sum file, I agree we should use it to verify what's on the disk for deps.

Reply:
`go mod tidy` and `go mod verify` added to `vendor` make target in 63cc9b77

Reply:
Since we're not overwriting `GOFLAGS`, are we then getting `-mod=vendor` as a flag on `go build` when it actually runs in CI?

Reply:
I assumed you we would update the Makefile `GO_BUILD_ARGS := -ldflags '-extldflags "-static"'` variable to use `-mod=vendor`.

Reply:
okay, done in to be pushed commit, waiting on CI tests to see results of `git diff` addition first.